classes/rnative.html

```diff
@@ -0,0 +1,52 @@
+---
+layout: default
+title: "Introduction to React Native"
+permalink: /classes/rnative
+---
+<h1 class="title page-title">Introduction to React Native</h1>
+<p class="title text-2x no-margin">Class & Curriculum Outline</p>
+<hr class="divider-rnative" />
+<p>Learn mobile web development with us! </p>
+
+<h2 class="title">
+    Brief Overview
+</h2>
+
+<p>Coming Soon!</p>
```

Comment:
For now, we could use:

> A year-long course that aims to give students a wide set of practical skills and project experience: students learn how to build websites with HTML, CSS, and Javascript, and then use those tools to build cross-platform mobile apps with React Native.

> All of our course materials are open-source! You can click into each lesson to see the notes that we use to teach the class. On top of that, we've put all of our code examples on GitHub; you can see them in the repo.

(there should be a link to the repo there)

> Have any questions? Feel free to reach out to Chase, our school lead, or anybody else on the Teach LA team!

(maybe link to Chase's page)

We probably want more formal content from @chasekap soon!